src/python_testing/matter_testing_support.py

```diff
@@ -1859,15 +1859,21 @@ def has_feature(cluster: ClusterObjects.ClusterObjectDescriptor, feature: IntFla
 
 
 async def get_accepted_endpoints_for_test(self: MatterBaseTest, accept_function: EndpointCheckFunction) -> list[uint]:
-    """ Helper function for the per_endpoint_test decorator.
+    """ Helper function for the run_for_each_matching_endpoint decorator.
 
         Returns a list of endpoints on which the test should be run given the accept_function for the test.
     """
     wildcard = await self.default_controller.Read(self.dut_node_id, [()])
```

Comment:
Would reading only the descriptor on each endpoint suffice? Would be a much faster read.

Reply:
yeah, probably would be faster. Thanks. Will change that.

Reply:
oh, no, wait, it needs to be able to determine the feature map and attribute list for each cluster as well. So I need the full wildcard.

Reply:
Maybe descriptor and globals would do though

Reply:
I'm going to land this change separately as a follow up PR since it requires a change to the Read function that I'd like to ensure gets proper feedback.